src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -662,7 +662,7 @@ protected function getContainerBuilder()
     {
         $container = new ContainerBuilder(new ParameterBag($this->getKernelParameters()));
 
-        if (class_exists('ProxyManager\Configuration')) {
+        if (class_exists('ProxyManager\Configuration') && class_exists('Symfony\Bridge\ProxyManager\LazyProxy\Instantiator\RuntimeInstantiator')) {
```

Comment:
I think not. This PR solves a case when "symfony/framework-bundle" and "ocramius/proxy-manager" are installed. The first condition solves a case when "symfony/symfony" is installed and "ocramius/proxy-manager" not.